compiler/test/dotty/tools/vulpix/SummaryReport.java

```diff
@@ -10,8 +10,10 @@
 import dotty.tools.dotc.reporting.TestReporter;
 import dotty.Properties;
 
-/** Note that while `ParallelTesting` runs in parallel, JUnit tests cannot with
- *  this class
+/** This class adds summary reports to `ParallelTesting`
+ *
+ *  It is written in Java because we currently cannot explicitly write static
+ *  methods in Scala without SIP-25 (`@static` fields and methods in Scala)
```

Comment:
If you only need static methods, isn't using a scala-defined object sufficient? The compiler will emit a corresponding class with static methods forwarding to the object's methods for interoperability with Java.